packages/admin/src/Pages/Actions/Concerns/CanBeDisable.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Filament\Pages\Actions\Concerns;
+
+use Closure;
+
+trait CanBeDisable
+{
+    protected bool | Closure $isDisabled = false;
+
+    public function disable(bool | Closure $condition = true): static
+    {
+        $this->isDisabled = $condition;
+
+        return $this;
+    }
+
+    public function isDisabled(): bool
+    {
+        return $this->isDisabled;
```

Comment:
```suggestion
        return value($this->isDisabled);
```

`isDisabled` can be a `Closure` so we need to evaluate it.